### DISCUSSION TITLE
How to install QModal (compilation fails)

### DISCUSSION THREAD

**Discussion by zvolsky** [quasarframework/quasar#11038] [General - Components / Directives / etc] [answered]
Hi friends.
Qv2.
I am not able install QModal.
When I add (in quasar.conf.js, section: framework) `components: ['QModal']` the compilation fails.
What now ...?

 App •  ERROR  •  UI  in ./.quasar/quasar-user-options.js

Error: QModal

  - import-transformation.js:6 module.exports    [frontend]/[quasar]/dist/transforms/import-transformation.js:6:11

  - loader.js.transform-quasar-imports.js:14     [frontend]/[@quasar]/app/lib/webpack/loader.js.transform-quasar-imports.js:14:78
  
  - Array.map
  
  - loader.js.transform-quasar-imports.js:10     [frontend]/[@quasar]/app/lib/webpack/loader.js.transform-quasar-imports.js:10:8
  
  - String.replace
  
  - loader.js.transform-quasar-imports.js:7 O    bject.module.exports
    [frontend]/[@quasar]/app/lib/webpack/load    er.js.transform-quasar-imports.js:7:30
  
 App •  COMPILATION FAILED  • Please check the log above for details.

**Comment by pdanpdan** [comment, marked as answer]
What is QModal?
:)

**Reply by zvolsky** [reply]
Yes. I'm stupid. I have found using Google some very old documentation.
Q-Dialog, of course.
